src/client/commands.ts

```diff
@@ -23,15 +27,15 @@ export function help() {}
  * Show version info.
  */
 export function ver() {
-	executeV('-v', (err: Error, stdout: string) => {
+	execV('-v', (err: Error, stdout: string) => {
```

Comment:
I don't think this is necessary to redefine the types and `err` is not `Error` type.

Reply:
After making a new type definition file. It would be better to change it.
```suggestion
	execV('-v', (err, stdout) => {
```